src/synced_services/toggl/toggl_synced_service.ts

```diff
@@ -405,35 +354,36 @@ export class TogglTrackSyncedService implements SyncedService {
       description: text,
       tags: tags,
       created_with: 'Timer2Ticket',
+      wid: this._serviceDefinition.config.workspace?.id,
     };
 
     const response = await this._retryAndWaitInCaseOfTooManyRequests(
       superagent
-        .post(this._timeEntriesUri)
+        .post(`${this._workspacesUri}/${this._serviceDefinition.config.workspace?.id}/time_entries`)
         .auth(this._serviceDefinition.apiKey, 'api_token')
-        .send({ time_entry: timeEntryBody })
+        .send({ timeEntryBody })
     );
 
     if (!response.ok) {
       return null;
     }
 
     return new TogglTimeEntry(
-      response.body.data['id'],
-      response.body.data['pid'],
-      response.body.data['description'],
-      new Date(response.body.data['start']),
-      new Date(response.body.data['stop']),
-      response.body.data['duration'] * 1000,
-      response.body.data['tags'],
-      new Date(response.body.data['at']),
+      response.body['id'],
+      response.body['pid'],
+      response.body['description'],
+      new Date(response.body['start']),
+      new Date(response.body['stop']),
+      response.body['duration'] * 1000,
+      response.body['tags'],
+      new Date(response.body['at']),
```

Comment:
Tu ide zrejme len o nespravne pouzitie kniznice v minulosti? Ak ano a toto je spravny pristup, tak prosim fixnut aj na 187 riadku u updateProject, aby bol v ramci celeho projektu jednotny pristup.

Reply:
Zde se jedná o to, že v původní verzi byla navrácená data obsažena v elementu "data", v nynější verzi jsou přímo v body

Reply:
U updateProject jsem původní `.data` přehlédl, díky za upozornění, opraveno

Reply:
Super, vdaka za si upravil aj to data 👍